packages/core/src/modules/sd-jwt-vc/SdJwtVcService.ts

```diff
@@ -393,10 +399,22 @@ export class SdJwtVcService {
       }
     }
 
-    throw new SdJwtVcError("Unsupported credential issuer. Only 'did' is supported at the moment.")
+    if (issuer.method === 'x5c') {
+      const leafCertificate = X509Service.getLeafCertificate(issuer.chain)
+
+      return {
+        alg: getJwkFromKey(leafCertificate.publicKey).supportedSignatureAlgorithms[0],
```

Comment:
We should check if the index 0 exists (could be undefined). Maybe we can add a `firstSupportedSignatureAlgorithm` that throws if not found? That way we don't have to do the check everywhere